default.json

```diff
@@ -10,8 +10,10 @@
     ":separateMultipleMajorReleases",
     "helpers:pinGitHubActionDigests"
   ],
+  "schedule": ["* 0-4 * * *"],
   "lockFileMaintenance": {
-    "enabled": true
+    "enabled": true,
+    "schedule": ["* 0-4 * * 1"]
```

Comment:
One weekly run, early Monday morning when no one should be working. We could really afford to be behind on some patch version of a 7th degree dependency.